project/Settings.scala

```diff
@@ -66,7 +66,8 @@ object Settings {
     usePgpKeyHex(sys.env.getOrElse("GPG_SIGNING_KEY", "0")),
     pgpPassphrase := sys.env.get("GPG_PASSPHRASE") map (_.toArray),
     pgpSecretRing := file(sys.env.getOrElse("GPG_PRIVATE_KEY", "")),
-    pgpPublicRing := file(sys.env.getOrElse("GPG_PUBLIC_KEY", ""))
+    pgpPublicRing := file(sys.env.getOrElse("GPG_PUBLIC_KEY", "")),
+    useGpg := false // still relies on BouncyCastle (https://github.com/sbt/sbt-pgp#usage)
```

Comment:
So, since the Bouncy Castle mode seems to be deprecated, should we maybe give the `gpg` mode a try? :)

@evbruno, from [here](https://github.com/sbt/sbt-pgp#install-gnupg-or-gnu-privary-guard-gpg), can you tell if it's going to work as is when running the release on Jenkins?